front/lib/api/assistant/actions/visualization.ts

```diff
@@ -105,15 +104,9 @@ export class VisualizationConfigurationServerRunner extends BaseActionConfigurat
     return new Ok({
       name,
       description:
-        description || "Generate graphs to visually represent the data.",
-      inputs: [
-        {
-          name: "query",
-          description:
-            "The query specifying the data visualization to generate.",
-          type: "string",
-        },
-      ],
+        description ||
```

Comment:
Unrelated, I only reworked the description and remove the query param since we don't use them anymore.